src/selectors/index.js

```diff
@@ -1,17 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+export * from './app';
```

Comment:
Here's where the magic happens. Flow is able to jump to the definition through this mechanism, which previously I think it hadn't.